packages/cip30/src/BrowserExtension/uiWallet.ts

```diff
@@ -1,10 +1,16 @@
 import { Logger, dummyLogger } from 'ts-log';
 import { Message } from './types';
 import { WalletApi } from '..';
-import { sendMessage } from './sendMessage';
+import { createMessenger } from './sendMessage';
 
-export const createUiWallet = (logger: Logger = dummyLogger): WalletApi => {
-  const methodNames = [
+export interface CreateUiWalletProps {
+  walletExtensionId?: string;
+  logger?: Logger;
+}
+
+export const createUiWallet = ({ logger = dummyLogger, walletExtensionId }: CreateUiWalletProps = {}): WalletApi => {
```

Comment:
```suggestion
export const createWalletApiMessageClient = ({ logger = dummyLogger, walletExtensionId }: CreateUiWalletProps = {}): WalletApi => {
```

+ rename this file to `walletApiMessageClient.tx`

We need more specificity here to communicate the purpose.